### PR TITLE
2021 03 16 deeamon128 html stuff

### PR DESCRIPTION
**Pull request by deeamon128**
added an header, footer, and navigation menu

### CODE REVIEW

**Review by adelacecilia — APPROVED** [review]
Cool! :)

**Review by BogdanNegura — APPROVED** [review]
👍

**Comment by deeamon128** [comment]
> added an header, footer, and navigation menu

- uploaded a font from google font
- javascript button
- box display

**Review by Constantin-Matei — APPROVED** [review]
Lasa te rog o linie libera dupa ultima linie de cod :)

**Review by andreimaff — APPROVED** [review]
💯

File: web/deeamon128/html/about-Jamal/index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <p>Hi there. My name is Andreea. I made this website to tell you about the adventures of Jamal in London</p>
+    <h3 style="font-family: cursive ;">Categories</h3>
+    <ul>
+      <li><a href="">Daily Adventures</a></li>
+      <li><a href="">Tips</a></li>
+      <li><a href="">Gallery</a></li>
+    </ul>
+  </nav>
+  <div class="box-shadow"> 
+    <a href='https://postimg.cc/8jtwpyF9' target='_blank'><img src='https://i.postimg.cc/8jtwpyF9/IMG-20200405-100957.jpg' border='0' alt='IMG-20200405-100957'/></a>
```
Comment (robertoalx) [inline web/deeamon128/html/about-Jamal/index.html]: If you used everywhere double quotes, you should use them on line 27 as well :)

File: web/deeamon128/html/about-Jamal/index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    <p id="demo" style="font-size: medium; border:0">Learn about Jamal nicknames</p>
+    <script>
+    function myFunction() { 
+    document.getElementById("demo").innerHTML = "Jammy Jam, Bula de Blana, Handsome Boi!";
+    }
+    </script>
+  </div>
+  <footer> @copyright Hamsters are cool 2021</footer>
+</body>
+</html>
```
Comment (robertoalx) [inline web/deeamon128/html/about-Jamal/index.html]: don't forget to add an empty linee

File: web/deeamon128/html/about-Jamal/style.css [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+.button__style {
+  border-radius: 10px;
+  border: 0;
+  background-color: hsl(341, 28%, 72%);
+}
+
+ul {
+  padding: 0 15px;
+  line-height: 25px;
+}
```
Comment (robertoalx) [inline web/deeamon128/html/about-Jamal/style.css]: Same here

**Review by robertoalx — APPROVED** [review]
Nice 💯

**Review by biggicode — APPROVED** [review]

**Review by biggicode — APPROVED** [review]

**Review by psyhedeliq — APPROVED** [review]

File: web/deeamon128/html/about-Jamal/style.css (hunk 3 above)
Comment (deeamon128) [reply web/deeamon128/html/about-Jamal/style.css]: Thanks. Will change it now :)

**Comment by deeamon128** [comment]
> Lasa te rog o linie libera dupa ultima linie de cod :)

**Pull request merged**